apps/src/gamelab/spritelab/Spritelab.js

```diff
@@ -0,0 +1,218 @@
+var commands = require('./commands');
+import * as spriteUtils from './spriteUtils';
```

Comment:
Would it make more sense to call this spritelabCommands? I'm thinking that if we ever want to completely separate gamelab and spritelab in the future, we would want Spritelab.js to instead include things like 'reset' or 'run'

Reply:
Then perhaps make Spritelabe.js just have drawBackground, updateTitle, and reset

Reply:
👏